src/jvmMain/kotlin/com/charleskorn/kaml/YamlNodeReader.kt

```diff
@@ -19,13 +19,8 @@
 package com.charleskorn.kaml
```

Comment:
Could you please revert the formatting changes in this file? It makes it very difficult to see what's really changed and what is just a formatting change.